### ISSUE TITLE
port forwarding fails when service port is using a named target port

### ISSUE THREAD

**Issue by sgandon** [txn2/kubefwd#13]
When a service is defined using a named target port like this
```
apiVersion: v1
kind: Service
metadata:
  name: streamsoss
spec:
  ports:
  - name: http-port
    port: 80
    targetPort: service-port
```
The port forwarding fails whereas if the targetPort is an hardcoded integer it works.

**Comment by sgandon** [comment]
I have tries to clone and compile the project but I can't manage to compile it.
I am a noob in Go, any lead so I may contribute ?
```
go run ./cmd/kubefwd/kubefwd.go
cmd/kubefwd/kubefwd.go:23:2: cannot find package "github.com/spf13/cobra" in any of:
	/usr/local/Cellar/go/1.11.2/libexec/src/github.com/spf13/cobra (from $GOROOT)
	/Users/sgandon/git-repos/kubefwd/src/github.com/spf13/cobra (from $GOPATH)
cmd/kubefwd/kubefwd.go:24:2: cannot find package "github.com/txn2/kubefwd/cmd/kubefwd/services" in any of:
	/usr/local/Cellar/go/1.11.2/libexec/src/github.com/txn2/kubefwd/cmd/kubefwd/services (from $GOROOT)
	/Users/sgandon/git-repos/kubefwd/src/github.com/txn2/kubefwd/cmd/kubefwd/services (from $GOPATH)
```
It seems go does not look into the /vendor folder.

**Comment by cjimti** [comment]
@sgandon this should be resolved in [1.4.5](https://github.com/txn2/kubefwd/releases/tag/1.4.5)
